src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -82,8 +83,9 @@
     <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
     <link rel="stylesheet" href="css/pageguide.min.css"/>
     <link rel="stylesheet" href="css/app.css">
-    <link rel="stylesheet" href="css/jquery.dropdown.css"/>
+    <link rel="stylesheet" href="css/main.css">
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:500,900" rel="stylesheet">
```

Comment:
Doesn't look like this font is getting picked up by any of the elements.  I think we have a more specific style set in the old css.